Makefile

```diff
@@ -8,10 +8,10 @@ NDK_BUILD = ndk-build
 WGET      = wget
 
 BASEURL   = https://github.com/measurement-kit/measurement-kit/releases/download
-VERSION   = v0.1.2
+VERSION   = v0.2.0-beta
 TAG       =
 INPUT     = measurement_kit-jni-$(VERSION)$(TAG).tar.bz2
-OVERSION  = $(VERSION).2
+OVERSION  = $(VERSION)-android-1
```

Comment:
Do you think `measurement_kit-android-v0.2.0-beta-1` would be ok as well, @lorenzoPrimi?